src/document-converter.ts

```diff
@@ -182,10 +182,10 @@ export class DocumentConverter {
         });
   }
 
-  convertToJsModule(): Iterable<ConversionOutput> {
+  convertToJsModule(): ConversionResult|void {
     for (const exclude of this.conversionMetadata.excludes) {
       if (this.originalUrl.endsWith(exclude)) {
-        return [];
+        return;
```

Comment:
Would it make sense to return a no-op ConversionResult? You use `results.has()` as a gate above, and populating the map would prevent multiple visits to this method.